src/debug.c

```diff
@@ -776,6 +791,11 @@ NULL
     {
         server.active_expire_enabled = atoi(c->argv[2]->ptr);
         addReply(c,shared.ok);
+    } else if (!strcasecmp(c->argv[1]->ptr,"packed_threshold") &&
+    c->argc == 3)
```

Comment:
* maybe use `memtol` instead of `atoi`?
* i think we should error if it is set to a value larger than 4gb (only allow reducing it)
* i don't like the fact we write to a plain global, let's either add it to the server struct, or add an accessor function in quicklist.c (and the global will be static)
* also, this subcommand isn't included in the help
```suggestion
    } else if (!strcasecmp(c->argv[1]->ptr,"quicklist_packed_threshold") && c->argc == 3)
```